### DISCUSSION TITLE
左側のキーボードが反応しない(シリアルポートの認識がしない)

### DISCUSSION THREAD

**Discussion by TKambara2119** [Yowkees/keyball#395] [Q&A]
初めての自作キーボードで解決できそうにないのでご助力下さい。
![IMG_3297](https://github.com/Yowkees/keyball/assets/127959278/bc4b763f-065f-41fb-9bd7-f75f300abb09)

1.ProMicroとOLEDの取り付けまで行い、テストのファームウェアを入れたタイミングですべてのキーが反応していない状態でした。
(左側のみ)
2.1度目はファームウェアの書き込みができたのですが、RESETスイッチを2回押してもリセットされず認識されない状態です。
(REMAPのファームウェア書き込み時)
RESETスイッチを押すとPC側からシステム音はするため、RESETスイッチは認識しているようです。

3.OLEDには何も表示されていません。
![IMG_3298](https://github.com/Yowkees/keyball/assets/127959278/669d1f94-6216-491c-a053-ffbac4199998)
ダイオードやProMicroのはんだを付け直したりしましたが解消しない状態です。

4.すべてのキーソケットが反応しません。

解決方法のアドバイスを頂けますと幸いです。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

お写真を拝見しまして、原因が分かりました。
12Pピンソケットの差し込み位置がズレています。表面から見て黒枠内に収まる位置に差し込んでください。

取り外し時にピンソケットが破損してしまった場合、Keyballシリーズご購入者様限定で部品単位での販売も可能ですので、白銀ラボウェブショップよりお問合せください。

以上、お手数ですがご対応お願いいたします。

**Reply by TKambara2119** [reply]
ご回答頂きましてありがとうございます。
12Pピンソケットを外そうとして破損してしまいました。
Shirogane Labさんの問い合わせから購入依頼をお送り致しました。

PS.
右側のキーボード(トラックボール有り)は正常に組み立てて、動作しております。
